modules/about-administrator-perspective.adoc

```diff
@@ -0,0 +1,26 @@
+// Module included in the following assemblies:
+//
+// web_console/web-console-overview.adoc
+
+:_content-type: CONCEPT
+[id="about-administrator-perspective_{context}"]
+= About the Administrator perspective in the web console
+
+The {product-title} web console provides two perspectives; the *Administrator* perspective and the *Developer* perspective. The *Administrator* perspective enables you to view the cluster inventory, capacity, general and specific utilization information, and the stream of important events, all of which help you to simplify planning and troubleshooting tasks. Both project administrators and cluster administrators can view the *Administrator* perspective.
+
+Cluster administrators can also open an embedded command line terminal instance in the web console in OpenShift Container Platform 4.7 and later. 
+
+[NOTE]
+====
+The default web console perspective that is shown depends on the role of the user. The *Administrator* perspective is displayed by default if the user is recognized as an administrator.
+====
+
+The *Administrator* perspective provides workflows specific to administrator use cases, such as the ability to:
+
+* Manage workload, storage, networking, and cluster settings.
+* Install and manage Operators using Operator Hub.
```

Comment:
Consider "the Operator Hub." Articles shouldn't be omitted per the IBM Style Guide.